scripts/deploy.actions.js

```diff
@@ -50,7 +50,6 @@ function deployActionsSync () {
   // for now this is a tmp hack so that ~/.wskprops does not interfer with WHISK_* properties defined in .env
   const fakeWskProps = '.fake-wskprops'
   fs.writeFileSync(fakeWskProps, '')
-
   process.env['WSK_CONFIG_FILE'] = fakeWskProps
   // aio reads env WHISK_* properties
   const aio = spawn.sync(
```

Comment:
@moritzraho Doesn't spawn.sync throw an error if the command exists with non zero error ?
Do we have to explicitly check and throw again asin line 66 and 67

Reply:
@rajarju yes pretty sure we have to. To be sure I tried to inject an error in the command and remove 66 and 67 and the error wasn't catch.